lib/logger/lib/logger/config.ex

```diff
@@ -132,6 +132,6 @@ defmodule Logger.Config do
     {:ok, %{config: data}} = :logger.get_handler_config(Logger)
     translators = fun.(data.translators)
     Application.put_env(:logger, :translators, translators)
-    :ok = :logger.set_handler_config(Logger, :config, %{data | translators: translators})
+    :ok = :logger.update_handler_config(Logger, :config, %{translators: translators})
```

Comment:
Unfortunately this will fail, as this is later addition to the API, OTP 21.0 do not have this function. This was my first approach as well.

Reply:
Thanks, I will revert it.